src/components/PlayerCard/PlayerCard.js

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+import styles from './PlayerCard.scss';
+
+const PlayerCard = ({
+  playerName,
+  ready = false,
+  button = null,
```

Comment:
I think this component has too much specific logic in here. This should probably be taken out and should be more composable. I think readying, the button, etc. is too much specific to this use-case. This should be done when consuming this component.